keyboards/lets_split/keymaps/geripgeri/keymap.c

```diff
@@ -0,0 +1,438 @@
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+	rgb_mode = RGBLIGHT_MODE_BREATHING + 1;
+	autoshift_enable();
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case ADJUST:
+      if (record->event.pressed) {
+        layer_on(_ADJUST);
+      } else {
+        layer_off(_ADJUST);
+      }
+      return false;
+      break;
+    case GAMING:
+      if (record->event.pressed) {
+	rgb_mode = RGBLIGHT_MODE_RAINBOW_SWIRL + 5;
+        autoshift_disable();
+	default_layer_set(1UL<<_GAMING);
+      }
+      return false;
+      break;
+    case NUMPAD:
+      if(record->event.pressed){
+	layer_on(_NUMPAD);
+	PORTB &= ~(1<<0);
+      } else {
+	layer_off(_NUMPAD);
+	PORTB |= (1<<0);
+      }
+      return false;
+      break;
+    case AE:           
+      if(record->event.pressed) {
+	SEND_STRING(SS_RALT("'"));
+      }    
+      break;
+    case EE:            
+      if(record->event.pressed) {
+	SEND_STRING(SS_RALT(";"));
+      }    
+      break;
+    case IE:            
+      if(record->event.pressed) {
+	SEND_STRING(SS_RALT("z"));
+      }    
+      break;
+    case VAI_NEE:            
+      if(record->event.pressed) {
+        rgblight_increase_val_noeeprom();
+      }    
+      break;
+    case VAD_NEE:            
+      if(record->event.pressed) {
+        rgblight_decrease_val_noeeprom();
+      }    
+      break;
+    case SAI_NEE:            
+      if(record->event.pressed) {
+        rgblight_increase_sat_noeeprom();
+      }    
+      break;
+    case SAD_NEE:            
+      if(record->event.pressed) {
+        rgblight_decrease_sat_noeeprom();
+      }    
+      break;
+    case HUI_NEE:            
+      if(record->event.pressed) {
+        rgblight_increase_hue_noeeprom();
+      }    
+      break;
+    case HUD_NEE:            
+      if(record->event.pressed) {
+        rgblight_decrease_hue_noeeprom();
+      }    
+      break;
+    case TOG_NEE:            
+      if(record->event.pressed) {
+	rgblight_toggle_noeeprom();
+      }    
+      break;
+    case RGB_ANM:            
+      if(record->event.pressed) {
+        rgblight_step_noeeprom();
+	rgb_mode = rgblight_get_mode();
+      }    
+      break;
+    case RGB_PRE:            
+      if(record->event.pressed) {
+        rgblight_step_reverse_noeeprom();
+	rgb_mode = rgblight_get_mode();
+      }    
+      break;
```

Comment:
Actually, if you use the normal keycodes here, and return false, it will override the normal behavior, without having to add new keycode.  I suspect that's what you are trying to do, here.

Reply:
Could you please verify that the way I implemented this does **not** use EEPROM for RGB at all?

Reply:
If you use the existing keycodes, and return false for them (which you appear to be doing), then yeah, it will not write to eeprom.

Though, you can verify that by enabling the console, as it should actually print a message if it writes to eeprom, from the functions. 

But from what I saw, it looks good.